src/components/widget/Lookup/Lookup.js

```diff
@@ -249,6 +280,19 @@ class Lookup extends Component {
     const errorInputCondition =
       validStatus && (!validStatus.valid && !validStatus.initialValue);
     const classRank = rank || 'primary';
+    let showBarcodeScannerBtn = false;
+
+    if (scanning) {
+      return (
+        <div
+          className="overlay-field js-not-unselect"
+          onKeyDown={e => e}
```

Comment:
That looks like you forgot to remove 😄

Reply:
Yeah, I'm not really sure what I was thinking here.